### ISSUE TITLE
Some training problems.

### ISSUE THREAD

**Issue by lhwwhl** [kaist-ina/NAS_public#6]
Dear Hyunho Yeo:

         I have read your article carefully recently and think this is a great project! In the process of learning, I have encountered some problems.
	1. Start training from a content-agnostic model (as described in this article). Does this content-agnostic model refer to using the provided DNN model with DIV2K training set?
	2. When can you open source dash.js module? I have a question, how does dash.js call pytorch trained models in the browser?

Looking forward to your reply!

**Comment by jaykim305** [comment]
Hi, I am one of the collaborators and am happy to answer your questions on behalf of Hyunho.

1. Yes. The content-agnostic model from our paper is trained with DIV2K dataset (https://data.vision.ee.ethz.ch/cvl/DIV2K/).
   Especially we used bicubic downsampled trainset of each scale.

2. Unfortunately, we can't open our dash.js source code for now because of our on going research. Sorry :(
 And, to answer your question, we didn't integrate pytorch to dash.js browser. 
We implemented seperate flask server with pytorch that handles request and runs DNN processing.
